src/platform/user/authentication/components/LoginActions.jsx

```diff
@@ -33,38 +35,79 @@ export default function LoginActions({ externalApplication }) {
       ? OCC_MOBILE.REGISTERED_APPS
       : OCC_MOBILE.DEFAULT;
 
+  const { useToggleValue, TOGGLE_NAMES } = useFeatureToggle();
+  const isSignInV2 = useToggleValue(TOGGLE_NAMES.signInModalV2);
+
+  const v2SignInProviders =
+    isSignInV2 || externalApplication?.includes(EXTERNAL_APPS.ARP)
+      ? allowedSignInProviders
+      : { ...allowedSignInProviders, dslogon: true, mhv: true };
+  const isValid = isRegisteredApp !== OCC_MOBILE.REGISTERED_APPS;
+
   return (
     <div className="row">
       <div className="columns small-12" id="sign-in-wrapper">
-        {reduceAllowedProviders(allowedSignInProviders, isRegisteredApp).map(
-          csp => (
-            <LoginButton csp={csp} key={csp} useOAuth={useOAuth} />
-          ),
-        )}
-        <LoginNote />
-        {externalApplication?.includes(EXTERNAL_APPS.VA_OCC_MOBILE) &&
-        isRegisteredApp === OCC_MOBILE.REGISTERED_APPS ? null : (
-          <div id="create-account">
-            <h2 className="vads-u-margin-top--3">Or create an account</h2>
-            <div className="vads-u-display--flex vads-u-flex-direction--column">
-              {reduceAllowedProviders(
-                allowedSignUpProviders,
-                externalApplication?.includes(EXTERNAL_APPS.VA_OCC_MOBILE) &&
-                isRegisteredApp === OCC_MOBILE.DEFAULT
-                  ? OCC_MOBILE.DEFAULT
-                  : '',
-              ).map(policy => (
-                <CreateAccountLink
-                  key={policy}
-                  policy={policy}
-                  useOAuth={useOAuth}
-                  externalApplication={externalApplication}
-                />
-              ))}
+        {reduceAllowedProviders(v2SignInProviders, isRegisteredApp).map(csp => (
+          <LoginButton csp={csp} key={csp} useOAuth={useOAuth} />
+        ))}
+        <LoginNote isV2={isSignInV2} />
+        {isSignInV2 &&
+          isValid && (
+            <div>
+              <h2>Other sign-in options</h2>
+              <h3 className="vads-u-margin--0">My HealtheVet sign-in option</h3>
+              <p>
+                <strong>Available through January 31, 2025</strong>
+              </p>
```

Comment:
I believe this should be a part of the `h3`. Probably best to add it as a `<span className="vads-u-display--block">` to get it to wrap appropriately.